packages/client/src/components/DishPopup/Ingredients.tsx

```diff
@@ -0,0 +1,98 @@
+import { Typography } from '@mui/material';
+import Box from '@mui/material/Box';
+import Checkbox from '@mui/material/Checkbox';
+import FormControl from '@mui/material/FormControl';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import FormGroup from '@mui/material/FormGroup';
+import * as React from 'react';
+
+const Ingredients = () => {
+  const [state, setState] = React.useState({
+    ingredient1: false,
+    ingredient2: false,
+    ingredient3: false,
+    ingredient4: false,
+    ingredient5: false,
+    ingredient6: false,
+    ingredient7: false,
+    ingredient8: false,
+    ingredient9: false,
```

Comment:
Będzie wygodniej dać `bool[]`, wtedy możesz mieć `n` składników a nie 9 na sztywno.

Plus jak zrobisz to co wyżej to tu będziesz miał dostęp do `dish.ingredients`.